pkg/config/defaults.go

```diff
@@ -60,6 +60,7 @@ func setDefaults(cfgType string) {
 	viper.SetDefault("auth.remote_authenticator.default_user_group", "Viewers")
 	viper.SetDefault("auth.remote_authenticator.request_timeout", 10*time.Second)
 
+	viper.SetDefault("blockstore.ensure_root_namespace_rw_access", true)
```

Comment:
And also update docs/reference/configuration.md, please :-/

Reply:
done